gflownet/envs/lattice_parameters.py

```diff
@@ -169,23 +165,27 @@ def get_actions_space(self) -> List[Tuple[int]]:
             - increment all alpha, beta and gama by n (required by rhombohedral lattice systems, for which
                 alpha == beta == gamma =/= 90 degrees).
         """
-        valid_steplens = np.arange(self.min_step_len, self.max_step_len + 1)
+        valid_steplens = [el for el in range(1, self.max_increment + 1)]
         actions = []
 
         # lengths
         for r in valid_steplens:
             for dim in [0, 1, 2]:
-                actions.append((dim,) * r)
-            actions.append((0, 1) * r)
-            actions.append((0, 1, 2) * r)
+                action = [0] * 6
```

Comment:
```suggestion
                action = [0 for _ in range(6)
```
Because [list multiplications are evil](https://towardsdatascience.com/dont-use-python-s-list-multiplication-n-n-450c68d673a3)! (It should not matter here though)